src/main/java/school/lemon/changerequest/java/cafe/exceptions/LowTemperatureException.java

```diff
@@ -0,0 +1,15 @@
+package school.lemon.changerequest.java.cafe.exceptions;
+
+public class LowTemperatureException extends ClientException {
```

Comment:
Looks like `Low` and `High` temperature exceptions should be from the same hierarchy... You have duplicated code.